wallet/txrules/rules.go

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) 2016 The btcsuite developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+// Package txrules provides transaction rules that should be followed by
+// transaction authors for wide mempool acceptance and quick mining.
+package txrules
+
+import (
+	"errors"
+
+	"github.com/btcsuite/btcd/txscript"
+	"github.com/btcsuite/btcd/wire"
+	"github.com/btcsuite/btcutil"
+)
+
+// DefaultRelayFeePerKb is the default minimum relay fee policy for a mempool.
+const DefaultRelayFeePerKb btcutil.Amount = 1e3
```

Comment:
dcrwallet has per network/ticket constants which can be set, see https://github.com/decred/dcrwallet/blob/master/wallet/createtx.go#L128-L138